src/ralph/accounts/admin.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from django.contrib.auth.admin import UserAdmin
+from django.utils.translation import ugettext_lazy as _
+
+from ralph.accounts.models import RalphUser
+from ralph.admin import RalphAdmin, register
+
+
+@register(RalphUser)
+class RalphUserAdmin(UserAdmin, RalphAdmin):
+
+    fieldsets = (
+        (None, {
+            'fields': ('username', 'password')
+        }),
+        (_('Personal info'), {
+            'fields': ('first_name', 'last_name', 'email')
+        }),
+        (_('Permissions'), {
+            'fields': (
+                'is_active', 'is_staff', 'is_superuser', 'groups',
+                'user_permissions'
+            )
+        }),
+        (_('Important dates'), {
+            'fields': ('last_login', 'date_joined')
+        }),
+        (_('Profile'), {
```

Comment:
maybe we should split profile info (gender, country, city etc) from job info (profit center, cost center, department etc)?